src/main/groovy/com/palantir/jacoco/CoverageViolation.groovy

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2015 Palantir Technologies, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.jacoco
+
+/**
+ * A coverage violation observed for the given {@code clazz}.
+ */
+public class CoverageViolation {
+    String cls
```

Comment:
done

(in general, insisting in lexicographic may not be optimal; Google for example suggests that any order can be acceptable as long as there it has a sound rationale; in this case, the order was random -- which is not considered a sound rationale ;)   https://google-styleguide.googlecode.com/svn/trunk/javaguide.html#s3.4.2-class-member-ordering )